### ISSUE TITLE
virtualbox kernel drivers broken on latest master with latest kernel

### ISSUE THREAD

**Issue by mkg20001** [NixOS/nixpkgs#84586]
**Describe the bug**
Virtualbox kernel drivers don't build with latest kernel.

**To Reproduce**

**on latest master, channel hasn't updated yet**

1. `d=(import /home/maciej/Projekte/nix { configuration = {pkgs, ...}: { boot.kernelPackages = pkgs.linuxPackages_latest; virtualisation.virtualbox.host.enable = true; }; }).vm   
`
2. `:b d`

**Expected behavior**
Builds

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.5.13, NixOS, 20.09pre-git (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4`
 - channels(root): `"nixos-20.09pre218613.ae6bdcc5358"`
 - channels(maciej): `"nixpkgs-20.09pre218717.05f0934825c"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by fabianhjr** [comment]
Hi @B4dM4n, sorry for the duplicate bug report.

You may find some useful information in this other issue trackers:

Upstream tracker:
 - https://www.virtualbox.org/ticket/19241

Related to this issue:
 - https://bugs.gentoo.org/706800
 - https://bugs.gentoo.org/715726 [Patch available]

**Comment by B4dM4n** [comment]
Virtualbox 6.1.6 was released last week, which fixes the Linux 5.6 issues. I opened a PR with a fix for `master`, which needs to be merged into #82693 to fix #84768.
